Shokofin/Configuration/configPage.html

```diff
@@ -92,7 +92,7 @@ <h3>Metadata Settings</h3>
                         <legend>
                             <h3>Library Settings</h3>
                         </legend>
-                        <div class="fieldDescription verticalSection-extrabottompadding">"Automatically merge series that are spread across multiple folders" must be disabled in the <a href="#!/library.html">Library Settings</a> if the Show/Season grouping set to "Do not group Series into Seasons", otherwise it must be enabled. If you do not do this then the plugin will not work as expected.</div>
+                        <div class="fieldDescription verticalSection-extrabottompadding">If you wish to set the Series/Grouping setting below to "Don't group Series into Seasons", you must disable "Automatically merge series that are spread across multiple folders" in the settings for all Libraries that depend on Shokofin for their metadata.<br>On the other hand, if you want to have Series and Grouping be determined by Shoko, or TvDB/TMDB - you must enable the "Automatically merge series that are stored in separate folders" setting for all Libraries that use Shokofin for their metadata.<br>See the settings under each individual Library here:<a href="#!/library.html">Library Settings</a></div>
```

Comment:
Change `<br>` to `<br/>` and the second mention of the settings is still wrong. Otherwise it looks ok. :+1:

Reply:
Scratch the first part i wrote.